WPAndroid Integration.md

```diff
@@ -0,0 +1,102 @@
+### Summary
```

Comment:
I think this could benefit from an `h1` title:

```suggestion
# WordPress Android Integration Guide

### Summary
```

When looking at the doc from GitHub, it's not immediately clear what it is about because the file name is as prominent as the `h3` in the rendered markdown:

![image](https://user-images.githubusercontent.com/1218433/111249766-540c3280-8660-11eb-921e-025f1b3077b9.png)

Reply:
Addressed in 21e81ffd7b7a2ea738a9fe0611f82f085f860cf1.